magpy/lib/format_iaga02.py

```diff
@@ -269,7 +278,8 @@ def writeIAGA(datastream, filename, **kwargs):
     """
     Writing IAGA2002 format data.
     """
-
+    global MISSING_DATA
```

Comment:
The `global` keyword is only required if you want to assign a value within the function.  Please remove from this function (or add to the readIAGA function)